rhel/SPECS/pmm-server.spec

```diff
@@ -56,6 +59,7 @@ install -d %{buildroot}%{_datadir}/%{name}/landing-page/img
 cp -pav ./entrypoint.sh %{buildroot}%{_datadir}/%{name}/entrypoint.sh
 cp -pav ./landing-page/img/pmm-logo.svg %{buildroot}%{_datadir}/%{name}/landing-page/img/pmm-logo.svg
 cp -pav ./%{pmm_repo}-%{pmm_commit}/api/swagger %{buildroot}%{_datadir}/%{name}/swagger
+cp -pav installation-wizard-page/build %{buildroot}%{_datadir}/%{name}/installation-wizard-page
 rm -rf %{pmm_repo}-%{pmm_commit}
```

Comment:
My understanding is that this line, and the one above it, are meant to be together.

Basically, put the new line one up.